web/pgadmin/tools/erd/static/js/erd_tool/components/ERDTool.jsx

```diff
@@ -577,7 +581,7 @@ export default class ERDTool extends React.Component {
     this.closeOnSave = closeOnSave;
     if(this.state.current_file && !isSaveAs) {
       this.saveFile(this.state.current_file);
-    } else {
+    } else if (this.diagram.getNodesData().length > 0){ {
```

Comment:
same as above. why disable saving empty files.

Reply:
Same for this, As when we open ERD tol our save button is disabled but our shortcut for save is working so It looks like a bug.